addons-cluster/pulsar/values.yaml

```diff
@@ -8,7 +8,7 @@ commonAnnotations:
   resource.kubeblocks.io/ignore-constraint: "true"
 
 image:
-  registry: apecloud-registry.cn-zhangjiakou.cr.aliyuncs.com
+  registry: docker.io
 
```

Comment:
should also change pulsar addon's cmpv, to make the namespace configurable.

```
      images:
        bookies-recovery: {{.Values.image.registry}}/apecloud/pulsar:2.11.2
```

The namespace is hardcoded to "apecloud"